easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.6-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'Gdk-Pixbuf'
+version = '2.42.6'
+
+homepage = 'https://developer.gnome.org/gdk-pixbuf/stable/'
+description = """
+ The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
+ It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
+ was distributed as part of GTK+ 2 but it was split off into a separate package
+ in preparation for the change to GTK+ 3.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['c4a6b75b7ed8f58ca48da830b9fa00ed96d668d3ab4b1f723dcf902f78bde77f']
+
+builddependencies = [
+    ('Meson', '0.58.0'),
+    ('Ninja', '1.10.2'),
+    ('binutils', '2.36.1'),
+    ('pkg-config', '0.29.2'),
+    ('GObject-Introspection', '1.68.0'),
+]
+
+dependencies = [
+    ('GLib', '2.68.2'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('libpng', '1.6.37'),
+    ('LibTIFF', '4.2.0'),
+    ('X11', '20210518'),
+]
+
+configopts = "--buildtype=release --default-library=both "
+configopts += "-Dgio_sniffing=false -Dgir=true "
```

Comment:
From my failed build (but not the reason for the failure):
```
WARNING: Unknown options: "gir"
```

Reply:
Ah, this option has changed name to `introspection`: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/commit/50387389beda3df666b3ef80811353b35f6d59fd (and this change is between the 2.40.0 and this 2.42.6).

However, it defaults to building it. Otherwise we can explicitly enable it with `-Dintrospection=enabled`.